hack/test-cmd.sh

```diff
@@ -699,7 +699,7 @@ __EOF__
   # Command: autoscale rc "frontend"
   kubectl autoscale -f examples/guestbook/frontend-controller.yaml --save-config "${kube_flags[@]}" --max=2
   # Post-Condition: hpa "frontend" has configuration annotation
-  [[ "$(kubectl get hpa frontend -o yaml "${kube_flags[@]}" | grep kubectl.kubernetes.io/last-applied-configuration)" ]]
```

Comment:
Hm, when you put it like that it sounds a lot better. OK.

On Fri, Feb 19, 2016 at 3:58 PM, David Eads <notifications@github.com>
wrote:

> In hack/test-cmd.sh
> <https://github.com/kubernetes/kubernetes/pull/21272#discussion_r53535639>
> :
>
> > @@ -699,7 +699,7 @@ __EOF__
> >    # Command: autoscale rc "frontend"
> >    kubectl autoscale -f examples/guestbook/frontend-controller.yaml --save-config "${kube_flags[@]}" --max=2
> >    # Post-Condition: hpa "frontend" has configuration annotation
> > -  [[ "$(kubectl get hpa frontend -o yaml "${kube_flags[@]}" | grep kubectl.kubernetes.io/last-applied-configuration)" ]]
>
> kind.group seems unnatural to me but I guess it's too late to change?
>
> We talked about it for a while here: #14461 (comment)
> <https://github.com/kubernetes/kubernetes/pull/14461#issuecomment-147833111>.
> Groups as qualified domain names give natural and identifiable namespacing.
> In that light, having "resource.group.domain.name" is a natural
> qualification.
>
> This pull establishes all the necessary parts to even support your
> suggested "resource.version.group". Since that would be fully specified, it
> can be built on these primitives as completed by discovery (still on my
> wishlist, but not for 1.2).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/21272/files#r53535639>.
>